TLM/TLM/U/UConst.cs

```diff
@@ -1,9 +1,9 @@
 namespace TrafficManager.U {
     public static class UConst {
         /// <summary>Orange color for shortcut text.</summary>
-        public static readonly string SHORTCUT_TEXT_HEX = "#cc994c";
+        private const string SHORTCUT_TEXT_HEX = "#cc994c";
```

Comment:
I'd have done those values static readonly because its not like these are values that can never change.
But for all intends and purposes it should not matter for us. if it breaks another mod in the future that depends on TM:PE, no big deal ^^